pallets/parachain-staking/src/lib.rs

```diff
@@ -968,18 +1000,20 @@ pub mod pallet {
 
 			Self::remove_candidate(&collator, &state)?;
 
-			if candidates
+			let (num_collators, num_delegators) = if candidates
```

Comment:
Same thing as above: why do we need to know `num_delegators`?

Reply:
Same argument as above.